sdk/identity/Azure.Identity/src/AzureCliCredentialClient.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Azure.Identity
+{
+    internal class AzureCliCredentialClient
+    {
+        public virtual (string, int) GetAzureCliAccesToken(string resource)
```

Comment:
This method should probably return the actual access token so that all the process execution and output parsing is handled in one place, rather than split between this class and `AzureCliCredential`

Reply:
Detail in this [PR](https://github.com/wantedfast/azure-sdk-for-net/pull/1)